flink-table/flink-table-api-scala-bridge/src/main/scala/org/apache/flink/table/api/bridge/scala/internal/StreamStatementSetImpl.scala

```diff
@@ -21,40 +21,39 @@ package org.apache.flink.table.api.bridge.scala.internal
 import org.apache.flink.annotation.Internal
 import org.apache.flink.table.api.bridge.scala.StreamStatementSet
 import org.apache.flink.table.api.internal.StatementSetImpl
-import org.apache.flink.table.api.{Table, TableDescriptor}
+import org.apache.flink.table.api.{ExplainDetail, Table, TableDescriptor, TablePipeline}
 
 /** Implementation for [[StreamStatementSet]]. */
 @Internal
 class StreamStatementSetImpl(tableEnvironment: StreamTableEnvironmentImpl)
     extends StatementSetImpl[StreamTableEnvironmentImpl](tableEnvironment)
     with StreamStatementSet {
 
-  override def addInsertSql(statement: String): StreamStatementSet = {
+  override def addInsertSql(statement: String): StreamStatementSet =
     super.addInsertSql(statement).asInstanceOf[StreamStatementSet]
-  }
```

Comment:
nit: IMHO we should leave curly braces for methods, we don't need to use very fancy Scala syntax, if you would add another statement, you would add the statement to the class init, pretty dangerous.